.github/workflows/release-npm.yml

```diff
@@ -43,29 +43,32 @@ jobs:
         with:
           ref: ${{ env.GIT_SHA }}
 
+      - name: Enable corepack
+        shell: bash
+        run: corepack enable
+
       - name: Set up Node.js ${{ env.NODE_VERSION }}
         uses: actions/setup-node@e33196f7422957bea03ed53f6fbb155025ffc7b8 # v3.7.0
         with:
           node-version: ${{ env.NODE_VERSION }}
-          cache: yarn
+          cache: pnpm
 
       - name: Init platform
         run: |
           git config --global core.autocrlf false
           git config --global core.symlinks true
           git config --global user.email 'renovate@whitesourcesoftware.com'
           git config --global user.name  'Renovate Bot'
-          yarn config set version-git-tag false
 
       - name: Installing dependencies
         uses: nick-fields/retry@943e742917ac94714d2f408a0e8320f2d1fcafcd # v2.8.3
         with:
           timeout_minutes: 10
           max_attempts: 3
-          command: yarn install --frozen-lockfile
+          command: pnpm install
```

Comment:
```suggestion
          command: pnpm install --frozen-lockfile
```

Reply:
not necessary, it's default on CI, like with yarn v2+ 🙃

Reply:
https://pnpm.io/cli/install#--frozen-lockfile